azure_monitor/src/azure_monitor/trace.py

```diff
@@ -134,8 +134,16 @@ def span_to_envelope(self, span):  # noqa pylint: disable=too-many-branches
                 data.type = "HTTP"  # TODO
```

Comment:
oh, would you mind fixing this one too in this PR? Or should I create another issue?

Reply:
I will fix this here! :)